### DISCUSSION TITLE
ofxhome.com dns error

### DISCUSSION THREAD

**Discussion by pallavg** [csingley/ofxtools#186] [General]
Is ofxhome.com no longer functional? I'm unable to use ofxget to pull statements now. Keep getting a dns error when trying to query ofxhome.

**Comment by csingley** [comment]
Yep, jseutter hung it up.  He was looking for anybody who wanted to maintain the site, but no takers.

**Reply by Anaxagoras-bc** [reply]
So does that effectively make this project dead?

**Reply by PrplHaz4** [reply]
I recognize there's a bit of a trust/safety issue with just handing over the site/data as a whole to an unknown.

Any chance he'd be willing to provide a data dump to this github org, at least freeing up the data?  Flat files per bank would be a great way to keep it alive with little to no maintenance required on the part of the owner.  Shame to have such a great resource lost.

**Reply by csingley** [reply]
I mean you could ask him.  I'm not personally well positioned to take on more OFX-related work these days, especially not with those dicks at FDX fuelling up on VC money to hold the pillow down tightly over its face - but if you were a credible maintainer I bet he'd talk to you.
